### ISSUE TITLE
Fix Bootstrap template

### ISSUE THREAD

**Issue by vanillajonathan** [dotnet/aspnetcore#44829]
## File a bug
Bootstrap 5 replaced the `form-group` class with `mb-3`.

### Include provider and version information

Microsoft.VisualStudio.Web.CodeGeneration.Designy version: 6.0.10
Target framework: .NET 6.0
Operating system: Windows 11
IDE: Visual Studio 2022 17.3.6

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
Can you please point to where exactly that recommendation is coming from?

**Comment by mkArtakMSFT** [comment]
We assume this is what you were referring to:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/34246760/213524539-f64640c2-1330-4c46-a536-f605d40a198d.png">
Source: https://getbootstrap.com/docs/5.0/migration/

**Comment by vanillajonathan** [comment]
@mkArtakMSFT Yes, that is what I was referring to. There might be other changes needed too. Label elements should the `form-label` class.
